.vscode/settings.json

```diff
@@ -11,5 +11,10 @@
     "go.buildFlags": [
         "-tags",
         "integration"
-    ]
+    ],
+    "terminal.integrated.env.linux": {
```

Comment:
I'm not sure what this does - should it be in this PR?

Reply:
oh probably not, let me remove it